tests/queries/0_stateless/02346_inverted_index_mutation.sql

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS t;
+
+set allow_experimental_inverted_index=1;
+CREATE TABLE t
+(
+    `timestamp` UInt64,
+    `s` String,
+    INDEX idx s TYPE inverted(3) GRANULARITY 1
+)
+ENGINE = MergeTree
+ORDER BY tuple()
+SETTINGS min_rows_for_wide_part = 1, min_bytes_for_wide_part = 1;
+
+INSERT INTO t (s) VALUES ('I am inverted');
+
+-- do update column
+ALTER TABLE t UPDATE s='I am not inverted' WHERE 1;
+
+-- wait for mutation to finish on at least 1 part
+SELECT sleepEachRow(1) FROM t LIMIT 1;
```

Comment:
It does not look reliable.

Reply:
is there some settings or statements for this ?

Reply:
I found `mutations_sync` to be a better way